src/client.js

```diff
@@ -109,6 +110,10 @@ export default function main() {
   const store = configureStore(initialState, {});
   context.store = store;
 
+  if (initialState.auth.isAuthenticated) {
+    store.dispatch(loadToken());
```

Comment:
Note, that this can trigger React *Warning: React attempted to reuse markup in a container but the checksum was invalid* because server renders without profile, but client loads profile *before* initial render.
This can be fixed by providing profile to server via cookie or by triggering this action after initial render.